catala/aixecar/guerrer/guerrer.md

```diff
@@ -0,0 +1,7 @@
+Et sents acorralat i penses que l'única forma de sortir d'aquí és transformarte en super guerrer...
+
+Respires profundament i comences a cridar com tants cops has vist per televisió...
+
+De sobte, un fort soroll et retorna a la realitat. Et despertes al llit de casa teva i te'n adones que t'has cagat a sobre!
+
+Clica [aquí](../catala.md) per tornar a començar...
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.